src/var/intervalClear.js

```diff
@@ -0,0 +1,4 @@
+define(function() {
+	/* global clearInterval: false */
+	return window.clearInterval || clearInterval;
```

Comment:
For which exactly `window.clearInterval` and just `clearInterval`? First is for jsdom and browser env, second for node?